wdl-analysis/tests/analysis/placeholder-options/source.diagnostics

```diff
@@ -0,0 +1,6 @@
+error: type mismatch for placeholder option `sep`: expected type `Array[P]` where P: any primitive type or Union, but found `Boolean`
```

Comment:
We'll need to regenerate the test baselines with `BLESS=1` with the above changes to `invalid_placeholder_option`.